docs/deploying-oobabooga.md

```diff
@@ -0,0 +1,58 @@
+# Deploying an Ooba instance
+
```

Comment:
I think we should just have the guides for the 1 click installers/docker on our wiki and leave the explanation of how to do it in python to their repo. Things may change for that setup in the future.